tls/extensions/s2n_client_psk.c

```diff
@@ -166,7 +166,8 @@ static int s2n_client_psk_send(struct s2n_connection *conn, struct s2n_stuffer *
         /* Write obfuscated ticket age */
```

Comment:
Thanks for including the list of changes made to all of the callbacks in your PR description. Looking at the list, it looks like no changes were made to `s2n_config_set_session_ticket_cb` (the `session_ticket_cb` callback), though I think we do want to change the check for this callback as well.